server/tools/attestation-service-ctl/readme.md

```diff
@@ -0,0 +1,34 @@
+# attestation-service-ctl
+
+attestation-service-ctl is a command line interface for the attestation server interface. It provides the following functions:
+- Set/Get each TEE's Open Policy Engine(OPA) `Policy(.rego)` and `Reference Data(.json)` files. The `Reference Data(.json)` contents should come from such as Reference Value Provider Service(RVPS), but this item is not reflected in this implementation.
+- Restore each TEE's `Policy(.rego)` and `Reference Data(.json)` to default value.
+- Provide the Attestation Server's `attestation` endpoint testing functionality.
+
+## Supported TEEs
+
+Currently the `attestation-service-ctl` supports the following types of TEE:
+- sgx
+- tdx
+- sevsnp
+- sample: The TEE used to demo/test Attestation Server's functionalities.
+
+## Usage
+
+Here are the steps of building and running of this `attestation-service-ctl`:
+
+### Build
+
+Build the attestation-service-ctl and Attestation Server.
+```shell
+git clone https://github.com/confidential-containers/attestation-service
+cd attestation-service
+cargo build --release
+```
+
+### Run
```

Comment:
Aside from how to run it, it would be helpful if this document also explained:

- _Who_ should run it.
-  Minimum privilege level it can be run with.
- _Where_ it should be run (can it connect remotely to another system?)